HW_05/hw_with_flask/blueprints/views.py

```diff
@@ -0,0 +1,37 @@
+from flask import Blueprint
+from webargs import fields
+from webargs.flaskparser import use_kwargs
+
+from HW_05.hw_with_flask.utils.database_handler import execute_query
+from HW_05.hw_with_flask.utils.return_cities_by_genre import return_cities_by_genre
+
+city_by_genre_blueprint = Blueprint("city_by_genre", __name__)
+
+
+@city_by_genre_blueprint.route("/get-city-by-genre")
+@use_kwargs({"genre": fields.Str(required=True)}, location="query")
+def get_city_by_genre(genre: str):
+    query = f"""
```

Comment:
Try to do this task only with SQL. So after SQL query you should just result on the frontend.